app/ns-ui-widgets-category/switch/code-behind/article.md

```diff
@@ -1,8 +1,4 @@
-In the following example is shown, how to create Switch via Code-behind
+In the following example is shown, how to create a Switch widget via the code-behind file.
```

Comment:
You should remove the ","or consider rephrasing the entire sentence to something like this:
`The following example shows how to create a Switch widget via the code-behind file.`